src/main/java/com/bitso/Bitso.java

```diff
@@ -176,6 +176,44 @@ public BitsoLedger getUserLedger(String... specificOperation){
         return new BitsoLedger(o);
     }
 
+    public BitsoWithdrawal getUserWithdrawals(String... withdrawalsIds){
+        String request = "/api/v3/withdrawals";
+        if(withdrawalsIds.length > 0){
+            request += "/";
+            for (String string : withdrawalsIds) {
```

Comment:
Que opinas si en vez de ponerle de mas y despues tener que cortar el string, haces:
`for (int i = 0; i < withdrawalsIds.length - 1; i++)`
y luego afuera del for:
`request += withdrawalsIds[withdrawalsIds.length]`